composer.json

```diff
@@ -17,7 +17,9 @@
         "WebSub"
     ],
     "require": {
-        "php": ">=7.0.0"
+        "php": ">=7.0.0",
+        "ext-pdo": "*",
+        "ext-simplexml": "*"
```

Comment:
This could better be in another PR, but that does not matter, and the list could be extended:
https://github.com/FreshRSS/FreshRSS/blob/8e398d24f132095fffba0db3c7ceea9a43371b4d/Docker/Dockerfile-Alpine#L7-L9